packages/traceloop-sdk/src/lib/tracing/span-processor.ts

```diff
@@ -44,6 +47,14 @@ export interface SpanProcessorOptions {
    * The headers to be sent with the traces data. Optional.
    */
   headers?: Record<string, string>;
+
+  /**
+   * The instrumentation libraries to be allowed in the traces. Optional.
```

Comment:
I'd have a slightly different behavior here - if the parameter is undefined - then it's all instrumentation libraries. If it's set - then whatever is set (which can either be an array of library names or TRACELOOP_INSTRUMENTATION_LIBRARIES) - wdyt?

Reply:
I thought of this option, it'll force users of the standalone processor to assign `TRACELOOP_INSTRUMENTATION_LIBRARIES` for `instrumentationLis` argument instead of having it by default

Reply:
it makes more sense tho imo, but has some more friction